src/pages/home/report/ReportActionItemGrouped.js

```diff
@@ -1,25 +1,26 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
-import styles from '@styles/styles';
+import stylePropTypes from '@styles/stylePropTypes';
+import useThemeStyles from '@styles/useThemeStyles';
 
 const propTypes = {
     /** Children view component for this action item */
     children: PropTypes.node.isRequired,
 
     /** Styles for the outermost View */
-    // eslint-disable-next-line react/forbid-prop-types
-    wrapperStyles: PropTypes.arrayOf(PropTypes.object),
+    wrapperStyle: stylePropTypes,
 };
 
 const defaultProps = {
-    wrapperStyles: [styles.chatItem],
+    wrapperStyle: {},
```

Comment:
```suggestion
    wrapperStyle: undefined,
```

Reply:
AFAIK, we don't want to use `undefined`. https://github.com/Expensify/App/pull/31774#discussion_r1404190805.

However, when I remove it from `defaultProps`, linter complaints, because `wrapperStyle` is not required.

Reply:
> AFAIK, we don't want to use undefined. https://github.com/Expensify/App/pull/31774#discussion_r1404190805.

@rezkiy37  That would be for TS files, for now I think you can use it and we will remove it when migrating.

Reply:
Done - ca70cac, 1d44d68.